archetypes/tasks.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+content_type: task
+---
+
+<!-- overview -->
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
`{{< version-check >}}` only really makes sense if `min-kubernetes-version:` is set in front matter.

Reply:
OK. I took this directly from a task page. I think it is a "default" message warning readers to check their version.
I am undecided.